lib/python/picongpu/picmi/simulation.py

```diff
@@ -107,7 +107,11 @@ def __init__(
 
         # additional PICMI stuff checks, @todo move to picmistandard, Brian Marre, 2024
         ## throw if both cfl & delta_t are set
-        if self.solver is not None and "Yee" == self.solver.method and isinstance(self.solver.grid, Cartesian3DGrid):
+        if (
+            self.solver is not None
+            and self.solver.method in ["Yee", "Lehe"]
```

Comment:
Introduce a named constant for this
```suggestion
            and self.solver.method in SOLVERS_WITH_YEELIKE_CFL_CONDITION
```
and also use it further below for the same list.